.gitignore

```diff
@@ -1,3 +1,2 @@
 *.swp
-Gemfile.lock
```

Comment:
It makes sense to me that we remove the Gemfile.lock. Without it we have no good way to replicate builds.